cli/lib/kontena/cli/containers/exec_command.rb

```diff
@@ -6,6 +6,8 @@ class ExecCommand < Kontena::Command
     include Kontena::Cli::GridOptions
     include Kontena::Cli::Helpers::ExecHelper
 
+    Clamp.allow_options_after_parameters = false
```

Comment:
Move these assignments outside of the class to be less deceptive?

Reply:
Tried something more fancy, yay or nay?

Reply:
Yet another solution

Reply:
Relying on the subcommand loader would be fine, I just meant that the code should be clearer as to what's going on.